src/components/NavBar.vue

```diff
@@ -31,6 +31,12 @@
             <span>Profile</span>
           </div>
         </div>
+        <div class="navbar-buttonWrapper desktop">
+          <button class="navbar-iconWrapper schedule-builder-icon full-opacity-on-hover" />
+          <div class="navbar-iconText">
+            <span>Schedule Builder</span>
```

Comment:
The text `Schedule Builder` here is currently too wide to fit in the base leftmost side panel, which causes it to stretch out past the bounds (and take with it the other text below icons on the sidebar).

Like you mentioned @nidhi-mylavarapu — alignment is wonky.

One way this could be fixed is to give this specific span element the following CSS properties:

1. `text-align: center`
2. `overflow-wrap: anywhere`

However note that the latter will break up the words into letters, and doesn't look too aesthetic. Need to consult designers.